.github/workflows/pr-add-dependabot-to-project.yaml

```diff
@@ -0,0 +1,104 @@
+on:
```

Comment:
Just saw we have no name set on those jobs, so the filepath + file name will be used as name and that doesn't look very nice.